seeg/sample_seeg_electrodes_fsaverage.tsv

```diff
@@ -0,0 +1,128 @@
+	name	R	A	S
+0	LACN 1	-7.0	14.5	25.0
```

Comment:
I still dream of a day we don't need this, but it's tiny so I'll merge so we can actually see what CircleCI thinks!